### DISCUSSION TITLE
Example Server does not show any tiles

### DISCUSSION THREAD

**Discussion by wielandb** [systemed/tilemaker#538] [Q&A]
I used tilemaker to generate a `.mbtiles` file, and everything went well. 
I now wanted to preview the tiles on the included ruby server, but no tiles show, just the red rectangles that I assume are placeholders for when there are no tiles. 
Having no tiles in some areas is expected for me, as I only generated a file for germany. I used the provided `example.json` config file and the provided `proccess.lua` for generating the tiles. Also, curiously, the tile server reports a 200 for all tiles that I try to load, even when scrolling to areas that should not be covered. 

I assume that when first loading the page, the map pans to an area that should be covered by the generated tiles.

What do I have to do to view my generated tiles?

**Comment by systemed** [comment]
Could you share the exact steps you took - i.e. where you downloaded the .osm.pbf from, the commands you issued, whether you altered the default config at all, etc.?

**Comment by wielandb** [comment]
Of course, here are the commands I issued:

```
wget https://download.geofabrik.de/europe/germany/baden-wuerttemberg-latest.osm.pbf
wget https://download.geofabrik.de/europe/germany/bayern-latest.osm.pbf
wget https://download.geofabrik.de/europe/germany/brandenburg-latest.osm.pbf
wget https://download.geofabrik.de/europe/germany/hessen-latest.osm.pbf
wget https://download.geofabrik.de/europe/germany/mecklenburg-vorpommern-latest.osm.pbf
wget https://download.geofabrik.de/europe/germany/niedersachsen-latest.osm.pbf
wget https://download.geofabrik.de/europe/germany/nordrhein-westfalen-latest.osm.pbf
wget https://download.geofabrik.de/europe/germany/sachsen-latest.osm.pbf
wget https://download.geofabrik.de/europe/germany/sachsen-anhalt-latest.osm.pbf
tilemaker --input baden-wuerttemberg-latest.osm.pbf --output out.mbtiles
tilemaker --input bayern-latest.osm.pbf --output out.mbtiles --merge
tilemaker --input brandenburg-latest.osm.pbf --output out.mbtiles --merge
tilemaker --input hessen-latest.osm.pbf --output out.mbtiles --merge
tilemaker --input mecklenburg-vorpommern-latest.osm.pbf --output out.mbtiles --merge
tilemaker --input niedersachsen-latest.osm.pbf --output out.mbtiles --merge
tilemaker --input nordrhein-westfalen-latest.osm.pbf --output out.mbtiles --merge
tilemaker --input sachsen-latest.osm.pbf --output out.mbtiles --merge
tilemaker --input sachsen-anhalt-latest.osm.pbf --output out.mbtiles --merge
cd path/to/tilemaker/server
ruby server.rb ~/tilemaker/out.mbtiles
```

My config file is [this one](https://github.com/systemed/tilemaker/blob/master/resources/config-example.json) and my lua-file is [this one](https://github.com/systemed/tilemaker/blob/master/resources/process-example.lua)

This is the output of the server command, despite seeing nothing, a 200 OK seems to be returned on every file:

```
127.0.0.1 - - [19/Sep/2023:19:54:48 CEST] "GET /14/8676/5477.pbf HTTP/1.1" 200 36
- -> /14/8676/5477.pbf
127.0.0.1 - - [19/Sep/2023:19:54:48 CEST] "GET /14/8676/5478.pbf HTTP/1.1" 200 36
- -> /14/8676/5478.pbf
127.0.0.1 - - [19/Sep/2023:19:54:48 CEST] "GET /14/8678/5478.pbf HTTP/1.1" 200 36
- -> /14/8678/5478.pbf
127.0.0.1 - - [19/Sep/2023:19:54:48 CEST] "GET /14/8678/5477.pbf HTTP/1.1" 200 36
- -> /14/8678/5477.pbf
127.0.0.1 - - [19/Sep/2023:19:54:48 CEST] "GET /14/8677/5478.pbf HTTP/1.1" 200 36
- -> /14/8677/5478.pbf
```

Here is what I am seeing:
![tilemaker](https://github.com/systemed/tilemaker/assets/29458682/693f5ab8-c51c-447b-8e80-9b92bd6f60a0)

**Comment by systemed** [comment]
Ah, I see. The `config-example.json` and `process-example.lua` files are just examples of how to structure these config files. They won't write much data to the tiles, and it won't be in the schema (=data layout) expected by the standard style file used by the Ruby server.

You should use `config-openmaptiles.json` and `process-openmaptiles.json`.
